bom-internal/pom.xml

```diff
@@ -141,9 +141,8 @@
             </dependency>
             <dependency>
                 <groupId>io.netty</groupId>
-                <artifactId>netty-transport-native-epoll</artifactId>
+                <artifactId>netty-transport-classes-epoll</artifactId>
```

Comment:
Motivated by build failure:
```
[INFO] --- maven-dependency-plugin:3.1.1:analyze-only (default) @ netty-nio-client ---
[WARNING] Used undeclared dependencies found:
[WARNING]    io.netty:netty-transport-classes-epoll:jar:4.1.72.Final:compile
[WARNING] Unused declared dependencies found:
[WARNING]    io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.72.Final:compile
```

Reply:
Is the dependency `netty-transport-classes-epoll` really useful without one version of `netty-transport-native-epoll` which contains the native library use by `netty-transport-classes-epoll`? Or are user expected to add their own dependency on `netty-transport-classes-epoll`?